code/Model/SiteTree.php

```diff
@@ -2100,8 +2100,10 @@ public function getSettingsFields()
             }
         }
 
+        $inheritMessage = $this->ParentID !== 0 ? _t(__CLASS__.'.INHERITPARENT', "Inherit from parent page") : _t(__CLASS__.'.INHERITSITECONFIG', "Inherit from site access settings");
```

Comment:
Just a pedantic comment about line breaks here, but this appeal to my sense of aesthetic more:
```php
        $inheritMessage = $this->ParentID !== 0 ? 
            _t(__CLASS__.'.INHERITPARENT', "Inherit from parent page") : 
            _t(__CLASS__.'.INHERITSITECONFIG', "Inherit from site access settings");
```

Reply:
Agree, I've updated to that format

Reply:
Is ParentID always an int? I thought it would normally be a string when pulled out of the DB...